compiler/src/dotty/tools/dotc/core/tasty/TastyPickler.scala

```diff
@@ -12,6 +12,15 @@ import core.Symbols.{Symbol, ClassSymbol}
 import ast.tpd
 import Decorators._
 
+object TastyPickler {
+
+  private val versionStringBytes = {
+    val compilerString = s"Scala compiler ${config.Properties.versionString}"
```

Comment:
Should this be the "minimum" compiler version, rather than the current one?

Reply:
No, I think the current one is the best choice.